report/sql_tasks/tasks/report/create_from_scratch/02_fdw/02_import_tables.jinja2.sql

```diff
@@ -0,0 +1,10 @@
+{% set lms_fdw_tables = [
+    ("public", "organization"), 
```

Comment:
```suggestion
    ("public", "organization"), 
    ("report", "organization_activity")
```

We'll want this pretty soon?